app/containers/RoomHeader/RoomHeader.tsx

```diff
@@ -151,6 +152,18 @@ const Header = React.memo(
 		const portrait = height > width;
 		let scale = 1;
 		const isMasterDetail = useAppSelector(state => state.app.isMasterDetail);
+		const statusState = useAppSelector(state => {
```

Comment:
This is just copy paste from Status component.
We should create a custom hook for this on `/lib` and use on both places.